traefik/values.yaml

```diff
@@ -1277,6 +1277,21 @@ hub:
         # -- Name of the header that will contain the tracestate copy.
         traceState: ""
 
+  # Define private plugin sources
+  pluginRegistry:
+    sources: {}
+      # plugin1:
+      # baseModuleName: "github.enterprise.local"
+      #   github:
+      #     entreprise:
+      #       url: "https://github.enterprise.local"
+      #     token: "urn:k8s:secret:github-token:access-token"
+      # plugin2:
+      # baseModuleName: "gitlab.company.com"
+      #   gitlab:
+      #     url: "https://gitlab.company.com"
+      #     token: "urn:k8s:secret:gitlab-token:access-token"
+
```

Comment:
```suggestion
```

In order to keep more shorter _values.yaml_, we may point to our doc or add an example. Wdyt ?